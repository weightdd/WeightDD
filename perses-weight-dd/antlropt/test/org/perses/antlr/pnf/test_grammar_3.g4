grammar test_grammar_3;

start
    : a b
    | b
    ;


a
    :
    ;

b
    : ID
    ;