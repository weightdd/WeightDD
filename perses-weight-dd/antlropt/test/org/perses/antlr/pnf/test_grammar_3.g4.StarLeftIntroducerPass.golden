// Auto-generated by Perses AST.
// DO NOT MODIFY.
grammar test_grammar_3;



start
    : a b
    | b
    ;

a
    : /* Epsilon. */
    ;

b
    : ID
    ;

